packages/hooks/src/stories/ConnectWallet.stories.tsx

```diff
@@ -1,16 +1,23 @@
-import React, { useEffect } from 'react';
 import { Button } from '@chakra-ui/react';
-import { NETWORKS, Provider, useWallet } from '..';
-import Authereum from 'authereum';
 import Portis from '@portis/web3';
+import Authereum from 'authereum';
+import React, { useEffect } from 'react';
+import { NETWORKS, Provider, useWallet } from '..';
```

Comment:
Why did you change the order of the imports? We should try to keep the changes to the minimum. Only introduce relevant changes so that it's easier for anyone to grasp the PR when re-visiting

Reply:
There is not any specific reason for that. I'll change it back. 😅